django/utils/datetime_safe.py

```diff
@@ -60,13 +60,10 @@ def new_datetime(d):
 def _findall(text, substr):
     # Also finds overlaps
     sites = []
-    i = 0
-    while 1:
-        j = text.find(substr, i)
-        if j == -1:
-            break
-        sites.append(j)
-        i = j + 1
+    i = text.find(substr, 0)
+    while i > -1:
+        sites.append(i)
+        i = text.find(substr, i + 1)
```

Comment:
While this duplicates, it seems like an improvement in readability since `j` is eliminated.